htmresearch/algorithms/column_pooler.py

```diff
@@ -110,19 +115,17 @@ def compute(self,
 
     @param  activeApicalCells (set)
             Indices of active cells that will form connections to apical
-            segments.
+            segments. Unused in this implementation.
 
     @param  formInternalConnections (bool)
             If True, cells will form
 
-    @param learn                If True, we are learning a new object
+    @param learn                    (bool)
+            If True, we are learning a new object
 
     """
     if activeExternalCells is None:
-      activeExternalCells = set()
-
-    if activeApicalCells is None:
-      activeApicalCells = set()
+      activeExternalCells = []
```

Comment:
FWIW the new TM/ETM doesn't use sets, so this awkwardness should go away after we finish updating the Python ETM.